pkg/volume/csi/csi_client.go

```diff
@@ -639,7 +639,7 @@ func (c *csiDriverClient) NodeGetVolumeStats(ctx context.Context, volID string,
 			metrics.Inodes = resource.NewQuantity(usage.GetTotal(), resource.BinarySI)
 			metrics.InodesUsed = resource.NewQuantity(usage.GetUsed(), resource.BinarySI)
 		default:
-			klog.Errorf("unknown key %s in usage", unit.String())
+			klog.ErrorS(nil, "unknown unit in VolumeUsage", "unit", unit.String())
```

Comment:
```
klog.ErrorS(fmt.Errorf("unknown unit in VolumeUsage"), "unknown unit in VolumeUsage", "unit", unit.String())
```

Reply:
No explicit error occurs in the switch-case code, and msg "unknown unit in VolumeUsage" already shows the error info, so I don't think it is neccesary to add fmt.Errorf("unknown unit in VolumeUsage"), what do you think of it?

Reply:
I agree with @CKchen0726; it seems to be redundant to add an error.

Reply:
This will result in "unknown unit in VolumeUsage" err= unit=xxxxx, if no error occured, may be this should be InfoS

Reply:
That‘s klog bug  [klog#216](https://github.com/kubernetes/klog/issues/216), here is good.